web/assets/css/production.less

```diff
@@ -232,6 +232,11 @@ li.prep-status {
   line-height: 28px;
 }
 
+.step-body .dictionary {
+  table-layout: fixed;
```

Comment:
What's this for? It seems to result in squishing the content a bit, since it tells the browser to keep the columns only as wide as the first row's columns, which can be short.

Reply:
I made the changes you suggested. Much better like this.